ufs2arco/regrid/regrid_ufs.py

```diff
@@ -0,0 +1,208 @@
+import yaml
+import numpy as np
+import xarray as xr
+from abc import ABC, abstractmethod
+
+from .gaussian_grid import gaussian_latitudes
+
+try:
+    import xesmf as xe
+except ImportError:
+    raise ImportError(
+        f"Cannot import xesmf. Install with 'conda install -c conda-forge xesmf',"
+        f" and note the environment must be deactivated and reactivated"
+    )
```

Comment:
Can we move the ImportError inside of RegridUFS.__init__? I think there are plenty of people who don't necessarily want to use the regridding capabilities and won't have installed xesmf. Moving the ImportError inside of the class makes it so that we only raise the error to relevant users.
```suggestion
try:
    import xesmf as xe
    _has_xesmf = True
except ImportError:
    _has_xesmf = False
```

Reply:
Ok thanks. Was that the reason the unit tests are failing?

Reply:
It is, even though it shouldn't cause them to fail. For some reason the windows runner has trouble installing xesmf, but it's hard to tell where the problem lies and how to debug it. This side steps the issue for now, which I think is reasonable